packages/web/app/tailwind.css

```
@@ -1,5 +1,10 @@
 @import 'tailwindcss';
 
+@plugin 'tailwind-scrollbar-hide';
+@import 'tw-animate-css';
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)